plans/ant/plan.sh

```diff
@@ -0,0 +1,26 @@
+pkg_origin=core
+pkg_name=ant
+pkg_version=1.9.7
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=(Apache 2)
+pkg_source=https://github.com/apache/ant/archive/rel/$pkg_version.tar.gz
+pkg_shasum=53dbfb13e0e9bf1b34bcbfdf807c1b9df1b89362b176e1e58150906f518b2c88
+pkg_deps=()
+pkg_build_deps=(core/jdk8 core/patchelf)
+pkg_bin_dirs=(bin)
+pkg_include_dirs=(include)
+pkg_lib_dirs=(lib)
+
+do_build() {
+  export JAVA_HOME=$(hab pkg path core/jdk8)
+  export LD_RUN_PATH=$LD_RUN_PATH:$(hab pkg path core/jdk8)/lib/amd64/jli
+  patchelf --interpreter "$(hab pkg path core/glibc)/lib/ld-linux-x86-64.so.2" \
```

Comment:
ah nope - https://github.com/habitat-sh/habitat/blob/master/plans/jdk8/plan.sh#L68
Looks like I missed the 'jre/bin`. I'd love to get that fixed up in the jdk8 plan.